web/lib/util/shortenedFromNow.ts

```diff
@@ -0,0 +1,26 @@
+import dayjs from 'dayjs'
+import duration from 'dayjs/plugin/duration'
+import {} from 'dayjs/plugin/duration'
+
+dayjs.extend(duration)
+
+export function shortenedFromNow(timeInPast: number): string {
+  const diff = dayjs.duration(dayjs().diff(timeInPast))
+
+  const units: { [key: string]: number } = {
```

Comment:
lol I just made something like this for the profile! Yours is probably better though 😄

Reply:
oh where is it maybe we can consolidate lol